examples/ci.json

```diff
@@ -17,10 +17,10 @@
     "NeuralNetwork": {
         "Architecture": {
             "model_type": "PNN",
-            "radius": 7,
+            "radius": 4,
```

Comment:
Is this size reduction mainly to save time?

Reply:
Yes